packages/ckb-sdk-core/src/index.ts

```diff
@@ -116,15 +135,35 @@ class Core {
     return this.config.secp256k1Dep
   }
 
+  public loadCells = async ({
```

Comment:
Is this for loading live (unspent) cells? If so can we rename it to `loadLiveCells` or `loadUnspentCells` for clarity?

Reply:
Found out that the underlying RPC method is `get_cells_by_lock_hash`. Then it seems just OK 😿